cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/SlackSelfHealingNotifier.java

```diff
@@ -66,13 +66,13 @@ public void alert(Object anomaly, boolean autoFixTriggered, long selfHealingStar
             return;
         }
 
-        String text;
+        String text =  String.format("%s detected %s. Self healing %s.", anomalyType, anomaly,
```

Comment:
Nit: Extra space after `=` and before `String`.

Reply:
Good call, fixed.